test/verdict_rails_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+require 'fake_app'
+
+class VerdictRailsTest < Minitest::Test
+
+    def setup
```

Comment:
The indentation on this file seems wrong. In Ruby we only use 2 spaces.

Reply:
good catch thank you!